### PR TITLE
Avoid reparse after initial render

### PR DESCRIPTION
**Pull request by mixonic**
Fixes https://github.com/bustlelabs/mobiledoc-kit/issues/260

### CODE REVIEW

**Comment by mixonic** [comment]
@bantic some quick eyes here would be great. Works well for me.

File: src/js/utils/compiler.js [hunk 1]
```diff
@@ -9,10 +11,15 @@ export function visit(visitor, node, opcodes) {
...
-    } else {
+    let length = params.length;
+    if (length === 0) {
       compiler[method].call(compiler);
+    } else if (length === 1) {
+      compiler[method].call(compiler, params[0]);
+    } else if (length === 2) {
+      compiler[method].call(compiler, params[0], params[1]);
+    } else {
+      compiler[method].apply(compiler, params);
```
Comment (bantic) [inline src/js/utils/compiler.js]: Curious if this is benchmarked? If so what's the speed-up?

**Comment by bantic** [comment]
:+1: looks good

File: src/js/utils/compiler.js (hunk 1 above)
Comment (mixonic) [inline src/js/utils/compiler.js]: Nah no serious benchmarks, lifted it from Ember where we use a similar pattern.

**Pull request merged**